models/Publication.js

```diff
@@ -32,6 +32,194 @@ const attributionTypes = [
   'translator'
 ]
 
+const languagesList = [
```

Comment:
pour prettifying tool likes to have one item per line in an array, apparently. 
Maybe I should move that to a separate file.